projects/github-actions/required-review/src/requirement.js

```diff
@@ -31,10 +32,13 @@ function buildReviewerFilter( config, teamConfig, indent ) {
 		const team = teamConfig;
 		return async function ( reviewers ) {
 			const members = await fetchTeamMembers( team );
-			return printSet(
+			teamReviewers = reviewers.filter( reviewer => members.includes( reviewer ) );
+			const neededTeams = teamReviewers.length ? [] : [team];
+			printSet(
 				`${ indent }Members of ${ team }:`,
-				reviewers.filter( reviewer => members.includes( reviewer ) )
+				teamReviewers
 			);
+			return {teamReviewers, neededTeams};
```

Comment:
You changed `printSet` so this will work:
```suggestion
			return printSet(
				`${ indent }Members of ${ team }:`,
				teamReviewers,
				neededTeams
			);
```